src/main/java/bio/terra/workspace/db/WorkspaceStageEnum.java

```diff
@@ -0,0 +1,58 @@
+package bio.terra.workspace.db;
+
+import bio.terra.workspace.db.exception.InvalidWorkspaceStageException;
+import bio.terra.workspace.generated.model.WorkspaceStageEnumModel;
+
+/**
+ * Internal representation of the workspace stage used for gating MC Terra features from Rawls
+ * workspaces. This is the enum serialized into the Workspace db table, not the enum generated as
+ * part of the external API. Current values RAWLS_WORKSPACE or MC_WORKSPACE.
+ */
+public enum WorkspaceStageEnum {
+  RAWLS_WORKSPACE("RAWLS_WORKSPACE_INTERNAL"),
+  MC_WORKSPACE("MC_WORKSPACE_INTERNAL");
```

Comment:
I don't think the string's need to be different for the internal representation. Just having the option to fork them later is good. If we do change them, we can change the enum mapping functions to map multiple enum values to a single enum of a different type.